okhttp-clients/src/test/java/com/palantir/conjure/java/okhttp/OkHttpClientsTest.java

```diff
@@ -111,6 +111,20 @@ public void cancelledCallsDoNotRetry() {
                 .satisfies(e -> assertThat(e.getCause()).hasMessage("Canceled").isInstanceOf(IOException.class));
     }
 
+    @Test
+    public void cancelledCallsDoNotReportToHostEventSink() {
+        assertThat(hostEventsSink.getMetrics()).isEmpty();
+
+        server.enqueue(new MockResponse().setHeadersDelay(1, TimeUnit.SECONDS).setBody("pong"));
+        OkHttpClient client = createRetryingClient(1);
+        AsyncRequest future = AsyncRequest.of(client.newCall(new Request.Builder().url(url).build()));
+        future.cancelCall();
+        assertThatExceptionOfType(UncheckedExecutionException.class).isThrownBy(() -> Futures.getUnchecked(future))
+                .satisfies(e -> assertThat(e.getCause()).hasMessage("Canceled").isInstanceOf(IOException.class));
+
+        assertThat(hostEventsSink.getMetrics()).isEmpty();
```

Comment:
think it's fine to stick this test in the above test and just call it `cancelledCallsDoNotRetryOrCountAsFailure` or whatever